Core/MIPS/IR/IRCompVFPU.cpp

```diff
@@ -1009,9 +1020,40 @@ namespace MIPSComp {
 	}
 
 	void IRFrontend::Comp_Vmscl(MIPSOpcode op) {
-		DISABLE;
+		CONDITIONAL_DISABLE;
+
+		// Matrix scale, matrix by scalar (no prefixes)
+		// d[N,M] = s[N,M] * t[0]
+
+		int vs = _VS;
+		int vd = _VD;
+		int vt = _VT;
+
+		MatrixSize sz = GetMtxSize(op);
+		if (sz != M_4x4) {
+			DISABLE;
+		}
+		if (GetMtx(vt) == GetMtx(vd)) {
+			DISABLE;
+		}
+		int n = GetMatrixSide(sz);
+
+		// The entire matrix is scaled equally, so transpose doesn't matter.  Let's normalize.
```

Comment:
I don't think this is right. What if one of them are transposed and the other is not? Then we need to transpose the output. If both are transposed, though, we can ignore it.